src/RoomInvite.js

```diff
@@ -63,10 +63,21 @@ export function showStartChatInviteDialog() {
 }
 
 export function showRoomInviteDialog(roomId) {
+    const cli = MatrixClientPeg.get();
+    const room = cli.getRoom(roomId);
```

Comment:
Could probably put the above two lines onto one line - `cli` is otherwise unused.